stylelint.config.js

```diff
@@ -0,0 +1,43 @@
+// Transform from snake-case to TitleCase
+const titleCase = str => str.replace(/(-|^)([^-]?)/g, (match, _, char) => char.toLowerCase());
```

Comment:
wait do we want this? why title case?

Reply:
The naming is off. It's actually setting a rule that allows the block to start out with a lower case of component name. 

Originally .modal would throw an error because it was expecting .Modal (to follow component and file naming). This module converts the expectation to be .modal as it should be.

Reply:
Really this regex is just taking the absolute first character and lower casing it ie try all caps

Reply:
Oh another edge case 'Hello-World' becomes 'helloworld' which is wrong

Reply:
You're right, it should be Pascal Case to kebab case. It'll reflect that in the recent push.